addon/components/bs-form.js

```diff
@@ -483,7 +483,7 @@ export default class Form extends Component {
     if (code === 13 && this.args.submitOnEnter) {
       let submitEvent = document.createEvent('Event');
       submitEvent.initEvent('submit', true, true);
-      event.target.dispatchEvent(submitEvent);
+      event.target.closest('form').dispatchEvent(submitEvent);
```

Comment:
I think we can just invoke `this.handleSubmit` here, can't we? This component catched the key press event bubbling up, so it is safe to assume that the component itself is the closest form, isn't it?